self-paced-labs/vertex-ai/vertex-ai-qwikstart/online-retail-clv-3M/requirements.txt

```diff
@@ -0,0 +1,3 @@
+explainable-ai-sdk==1.3.0
+tensorflow-io==0.16.0
+pyarrow
```

Comment:
Add these to the requirements file
wget
xlrd
openpyxl

Reply:
Thanks, adding these to the lab requirements.txt outside the container as these should just be dependencies for the data download script.